cve_bin_tool/csv2cve.py

```diff
@@ -1,241 +1,26 @@
-            if (all_cves < 0) or not all_cves:
-                return all_cves
-        except Exception:
-            self.output_engine = OutputEngine(
-                filename=outfile, products=all_cves, logger=self.logger
-            )
-            self.output_engine.output_file(format)
 
 
 def main(argv=None):
-    """ Take a list of product information + versions from a CSV file,
-    and output a list of matching CVES """
-
-    if argv is None:
-        argv = sys.argv
-
-    parser = argparse.ArgumentParser(
-        prog="csv2cve",
-        description="This tool takes a list of software + versions from a CSV file \
-        and outputs a list of CVEs known to affect those versions",
-    )
-    parser.add_argument(
-        "csv_file",
-        action="store",
-        help="CSV file with product data. Must contain vendor,product,version, \
-        where vendor and product match entries in the National Vulnerability Database.",
-    )
-    output_group = parser.add_argument_group("Output")
-    output_group.add_argument(
-        "-l",
-        "--log",
-        help="log level (default: info)",
-        dest="log_level",
-        action=LogAction,
-        choices=["debug", "info", "warning", "error", "critical"],
-    )
-    output_group.add_argument(
-        "-o",
-        "--output-file",
-        action="store",
-        default=None,
-        help="provide output filename (default: output to stdout)",
-    )
-    output_group.add_argument(
-        "-c",
-        "--cvss",
-        action="store",
-        default=0,
-        help="minimum CVSS score (as integer in range 0 to 10) to report (default: 0)",
-    )
-    output_group.add_argument(
-        "-S",
-        "--severity",
-        action="store",
-        choices=["low", "medium", "high", "critical"],
-        default="low",
-        help="minimum CVE severity to report (default: low)",
-    )
-    output_group.add_argument(
-        "-f",
-        "--format",
-        action="store",
-        choices=["csv", "json", "console"],
-        default="console",
-        help="update output format (default: console)",
-    )
-    output_group.add_argument(
-        "-q", "--quiet", action="store_true", help="suppress output"
-    )
-    parser.add_argument(
-        "-u",
-        "--update",
-        action="store",
-        choices=["now", "daily", "never", "latest"],
-        default="daily",
-        help="update schedule for NVD database (default: daily)",
-    )
-
-    if len(argv) <= 1:
-        parser.print_help()
-        return 0
-
-    args = parser.parse_args(argv[1:])
-    if int(args.cvss) > 0:
-        score = int(args.cvss)
-    elif args.severity:
-        # Set minimum CVSS score based on severity
-        cvss_score = {"low": 0, "medium": 4, "high": 7, "critical": 9}
-        score = cvss_score[args.severity]
+    argv = argv or sys.argv
+    if len(argv) < 2:
+        LOGGER.error("Invalid Usage: csv file required")
+        return -2
```

Comment:
If we're importing cli, we might as well use the same error code list here.

Reply:
I will change that